installation_and_upgrade/ibex_install_utils/tasks/backup_tasks.py

```diff
@@ -64,6 +65,27 @@ def backup_old_directories(self):
                 f"Unable to find data directory '{BACKUP_DATA_DIR}'. Please backup the current installation of IBEX "
                 f"manually")
 
+    #This function checks if the backup has been sucessful by checking for a VERSION.txt file within the backup folders
+    @task("Verify backup") 
+    def backup_checker(self):
+        """
+        Verify backup
+
+        """
+        EPICS_PATH_BACKUP = os.path.join(self._get_backup_dir(),'EPICS')
+        PYTHON3_PATH_BACKUP = os.path.join(self._get_backup_dir(),'Python3')
+        GUI_PATH_BACKUP = os.path.join(self._get_backup_dir(),'Client_E4')  
+
+        backup_paths = (EPICS_PATH_BACKUP, PYTHON3_PATH_BACKUP, GUI_PATH_BACKUP)
+
+        for d in backup_paths:
+            version_list = glob.glob(os.path.join(d, "VERSION.txt"))
```

Comment:
glob is normally used with a wildcard to match several files, maybe  `os.path.is_file()` is better here?

Reply:
We changed it back to os.path.exists()